text_localization_environment/TextLocEnv.py

```diff
@@ -120,6 +148,13 @@ def step(self, action):
     def calculate_reward(self, action):
         reward = 0
 
+        if self.has_termination_action:
+            if self.action_set[action] == self.terminate:
+                if self.pct_triggers_used != 1.0:
+                    return -10
+                else:
+                    return 10 + (self.current_step * self.DURATION_PENALTY)
+
         if self.action_set[action] == self.trigger:
             reward = 10 * self.ETA * self.iou - (self.current_step * self.DURATION_PENALTY)
```

Comment:
It bothers me that the reward for the trigger action is 70 * iou and the termination is 10 + small number * steps. Doesn't that make it harder to learn the termination action?

Reply:
The idea was to make the termination trigger not that attractive first compared to finding text instances. In previous configs I had 10 + sef.ETA2 [...] for the termination action (cf. config4 in evaluation google sheet) but the performance was worse than for the binary reward scheme